waspc/data/Generator/templates/sdk/wasp/server/utils.ts

```diff
@@ -2,12 +2,12 @@
 import { Request, Response, NextFunction } from 'express'
 
 {=# isAuthEnabled =}
-import { type AuthUser } from 'wasp/auth'
+import { type AuthUserData } from 'wasp/auth'
 {=/ isAuthEnabled =}
 
 type RequestWithExtraFields = Request & {
   {=# isAuthEnabled =}
-  user?: AuthUser;
+  user: AuthUserData | null;
```

Comment:
This is not really correct. The `user` can be `undefined` if there is no auth token present!

Reply:
I've adjusted the code to now always be `null` for non-authenticated user.